test_reflectable/pubspec.yaml

```diff
@@ -10,11 +10,11 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 dependencies:
   build_runner: any
-  glob: ^1.2.0
+  glob: any
```

Comment:
test_reflectable now uses `any` for all dependencies, such that all version constraints are taken from reflectable itself.